### DISCUSSION TITLE
How can I ignore the paths in my tsconfig.json file?

### DISCUSSION THREAD

**Discussion by sawa-ko** [swc-project/swc#7696] [Q&A] [answered]
I am currently using paths in my `tsconfig.json` file to resolve the typing of some imports of my `monorepo packages`, and then those packages I have installed in my package.json file, so I don't need their paths to be resolved since they point to their original `.ts` files and then when I compile the project I get path errors because they are being resolved to, for example: `src/index.ts`, and I don't need that.

How can I make swc not to resolve the paths that I have in my `tsconfig.json` file or that I can choose which ones to resolve and which ones not to resolve.

**Comment by kdy1** [comment, marked as answer]
You should create multiple config file and pass one to swc

**Reply by sawa-ko** [reply]
Wdym? An example please.

**Reply by kdy1** [reply]
- `tsconfig.json`
 - `tsconfig.dev.json`
